src/components/pages/HomePage.js

```diff
@@ -0,0 +1,311 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Menu, Modal, Input, Grid, Button, Form } from 'semantic-ui-react';
+import { Link } from 'react-router-dom'
+
+import requestPipeline from '../../components/pages/requestPipeline';
+
+import Styles from '../../styles/HomePage';
+import Actions from '../../reducers/Actions'
+import { statekeys } from '../../helpers/Common'
+
+export class HomePage extends Component {
+  render() {
+    const {setEmail, setPassword, setFirstName, setLastName, setPasswordRepeat, setAddress, setCity, setAddressState, setZipcode, openLoginModal, closeLoginModal, isLoginModalOpen, openSignupModal, closeSignupModal, isSignupModalOpen} = this.props;
+
+    return (
+            <div>
+            <div style={Styles.navbarContainer}>
+                <Menu secondary style={Styles.loginSignupMenu}>
+                    <Menu.Menu position='right'>
+                        <Modal
+      style={Styles.modal}
```

Comment:
We need to set up that eslint config. There's a lot of rough indenting.

Reply:
There is a story out there to do it, I even included the config file for eslint in the card itself on Trello.